modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js

```
@@ -49,6 +49,8 @@ import createResourceURL from './util/portlet_url/create_resource_url.es';
 import {getSessionValue, setSessionValue} from './util/session.es';
 import toCharCode from './util/to_char_code.es';
 
+Liferay = Liferay || {};
```

Does this actually do anything?

If `Liferay` is not defined, I would expect this to throw a `ReferenceError: Liferay is not defined`, and not fall back to the empty `{}` object. To have the effect that you seem to want here, @markocikos, it would have to be something like:

```javascript
Liferay = window.Liferay || {};
```

Or am I missing something?

---

In any case, just as a precaution, added https://github.com/wincent/liferay-portal/pull/140/commits/a40e5ffcc38f05867170d8db34edc575279f1622 on top.